api.go

```diff
@@ -983,13 +984,35 @@ func (lapi *Client) GetAsset(id string) (*Asset, error) {
 	return &asset, nil
 }
 
-func (lapi *Client) ListAssets() (*[]Asset, error) {
-	var assets []Asset
+func (lapi *Client) ListAssets() ([]*Asset, error) {
+	var assets []*Asset
 	url := fmt.Sprintf("%s/api/asset", lapi.chosenServer)
 	if err := lapi.getJSON(url, "asset", "", &assets); err != nil {
 		return nil, err
 	}
-	return &assets, nil
+	return assets, nil
+}
+
+func (lapi *Client) GetAssetByPlaybackID(pid string, includeDeleted bool) (*Asset, error) {
+	var assets []*Asset
+	url := fmt.Sprintf(`%s/api/asset?limit=2&allUsers=true&filters=[{"id":"playbackId","value":%q}]`, lapi.chosenServer, pid)
```

Comment:
Just curious on why specifically limit `2`? When is it possible that there are multiple assets with the same `playbackId`? I see this is also contemplated as a possible error later on line `1008`

Reply:
We only care about 3 scenarios:
 a. no results, which means we should 404
 b. 1 result, meaning we found the asset we want
 c. more than 1 result, which means there is an unexpected playback ID conflict

On the last case, we also don't care if we get 2 or 473263672 results, it only matters that there is more than 1.

In such case, we can limit the list to 2 elements in the response, which gives us only 3 possibilites of responses:
 - 0 elements (a)
 - 1 element (b)
 - 2 elements (c)

We also avoid some data bandwidth in case there is this bug of more than 1 asset with the same `playbackId`, transferring only 2 elements over the internet (doesn't matter how much there really are).

I don't know how that case could happen, but it is a safeguard since we're using a `List` function for what is actually a `Get`. If we get more than 1 asset, it's an internal server error and we should treat it as such.